basterdz-domain/src/main/java/com/dnd/room/Room.java

```diff
@@ -0,0 +1,47 @@
+package com.dnd.room;
+
+import com.dnd.common.entity.BaseTimeEntity;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+import static lombok.AccessLevel.PROTECTED;
+
+@Getter
+@Entity
+@NoArgsConstructor(access = PROTECTED)
+public class Room extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    @Column(name = "room_id")
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String resolution_message;
+
+    @Column(nullable = false)
+    private int maxHeadCount;
+
+    @Column(nullable = false)
+    private LocalDateTime start_date;
+
+    @Column(nullable = false)
+    private LocalDateTime end_date;
```

Comment:
```suggestion
    @Column(nullable = false)
    private LocalDateTime startDate;

    @Column(nullable = false)
    private LocalDateTime endDate;
```
SnakeCase로 하신 이유가 있을까요?